### PR TITLE
Aplazo icon

### PR DESCRIPTION
**Pull request by JuanGrassoConexa**
** Checklist **
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [x] I have provided a link to the brand icon’s brand guidelines whenever possible.
- [x] I recognize that if my icon is not approved by the Shopify Partners team it may not receive review nor merger.

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

**Comment by larouxn** [comment]
👋 @JuanGrassoConexa, can we adjust this new icon to match the same shape/size as Visa and other icons? We can probably use the [template](https://github.com/activemerchant/payment_icons/blob/62017ee4bd124d8dc0eea23868f5c49ee84f4aa6/CONTRIBUTING.md#template).

Screen Shot 2022-02-25 at 15 50 18

**Comment by JuanGrassoConexa** [comment]
Hi ther! @larouxn 
I just adjusted the icon so it match the required specifications.
I hope that now it's fine.

**Review by larouxn — CHANGES_REQUESTED** [review]
Looks like we need to add a `<title>` element to the SVG markup. ([ci logs](https://github.com/activemerchant/payment_icons/runs/5814806961?check_suite_focus=true))

```
/home/runner/work/payment_icons/payment_icons/vendor/bundle/ruby/3.1.0/gems/frozen_record-0.19.5/lib/frozen_record/base.rb:31: warning: ambiguous first argument; put parentheses or a space even after `-' operator
Run options: --seed 36558

# Running:

.........F

Failure:
PaymentIconTest#test_Every_payment_SVG_meets_accessibility_requirements [/home/runner/work/payment_icons/payment_icons/test/unit/payment_icon_test.rb:68]:
{:message=>"The 'aplazo' SVG file should have a single <title> tag"}.
Expected: 1
  Actual: 0

rails test /home/runner/work/payment_icons/payment_icons/test/unit/payment_icon_test.rb:61

...

Finished in 0.738983s, 17.5917 runs/s, 4809.3133 assertions/s.
13 runs, 3554 assertions, 1 failures, 0 errors, 0 skips
```

**Comment by JuanGrassoConexa** [comment]
Hi @larouxn ! Exactly what change is necessary to get this approved ?

**Comment by larouxn** [comment]
👋 Hey @JuanGrassoConexa, seeing the following error. ([CI logs](https://github.com/activemerchant/payment_icons/runs/6664411001?check_suite_focus=true))

> {:message=>"The 'aplazo' SVG file should have a 'role' attribute on the root <svg> tag"}

[Template](https://github.com/activemerchant/payment_icons/blob/5e3470d017d76f246b1270402351fe5c50fd602b/CONTRIBUTING.md#template) might help.

**Comment by larouxn** [comment]
Looks like the the `title` value is off. ([CI logs](https://github.com/activemerchant/payment_icons/runs/7052397420?check_suite_focus=true))

```
{:message=>"The 'aplazo' SVG file does not have the appropriate 'id' value on the <title> tag"}.
Expected: "pi-aplazo"
  Actual: "pi-Aplazo"
```

**Comment by larouxn** [comment]
👋 Hey @JuanGrassoConexa, can we update the icon to match the shape/structure of the other icons? ([template](https://github.com/activemerchant/payment_icons/blob/17bc4291a913d3bc178b78c25bbafff0b8733401/CONTRIBUTING.md#template))

Screen Shot 2022-07-07 at 10 05 39Screen Shot 2022-07-07 at 10 04 23

File: db/payment_icons.yml [hunk 1]
```diff
@@ -1227,13 +1227,13 @@
...
+  label: Aplazo
+
   name: paybylink
   label: Pay By Link
-  group: other
--
-  name: snap_checkout
-  label: Snap Checkout
-  group: other
+
```
Comment (larouxn) [inline db/payment_icons.yml:1236]: Something odd going on here. Can we rebase this PR to resolve this diff? Should not be removing `snap_checkout`.

**Review by larouxn — CHANGES_REQUESTED** [review]
One issue in `payment_icons.yml`. Additionally, looks like the `aria` label is incorrect.

```ruby
Failure:
PaymentIconTest#test_Every_payment_SVG_meets_accessibility_requirements [/home/runner/work/payment_icons/payment_icons/test/unit/payment_icon_test.rb:102]:
{:message=>"The 'aplazo' SVG file should have a aria-labelledby='pi-aplazo' attribute on the root <svg> node"}.
Expected: "pi-aplazo"
  Actual: "pi-{aplazo}"
rails test /home/runner/work/payment_icons/payment_icons/test/unit/payment_icon_test.rb:61
......
Finished in 1.223603s, 10.6244 runs/s, 6444.9028 assertions/s.
13 runs, 7886 assertions, 1 failures, 1 errors, 0 skips
```

**Review by larouxn — APPROVED** [review]
Looks good to me 🎩 
Screen Shot 2022-07-21 at 15 33 38

**Review by tauthomas01 — APPROVED** [review]

**Comment by JuanGrassoConexa** [comment]
Hi there, now that the changes has been approved, is anything left I should do? 

If not. When is the logo going to be available in Shopify?

Thanks!

**Comment by larouxn** [comment]
👋 Hey @JuanGrassoConexa, originally wanted to make this icon available this week but more likely it'll be available within next week.

**Comment by larouxn** [comment]
👋 Hello, we're working on the [v1.7.16](https://github.com/activemerchant/payment_icons/pull/702) release now. It will likely go live next week.

Apologies for the wait. Will report back here when it's live. 🙇‍♂️

**Comment by larouxn** [comment]
👋 Hello, your new/updated icon(s) should now be live everywhere. ✨

**Pull request merged**